ci/cleanup

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -eux
+
+rm -rf $1/.git
```

Comment:
As I understand, here are the files that we remove before comparing prod-2-7 to prod-2-7-adavance.

This desserve a description (`ci/cleanup` is not enough).